Dockerfile

```
@@ -1,14 +1,32 @@
+FROM ubuntu:24.04 AS chromaprint-build
+
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)